src/shell_minimal.js

```diff
@@ -5,9 +5,12 @@
  */
 
 #if USE_CLOSURE_COMPILER
-// if (!Module)` is crucial for Closure Compiler here as it will otherwise replace every `Module` occurrence with the object below
+// if (!Module)` is crucial for Closure Compiler here as it will
+// otherwise replace every `Module` occurrence with the object below
 var /** @type{Object} */ Module;
 if (!Module) /** @suppress{checkTypes}*/Module = {"__EMSCRIPTEN_PRIVATE_MODULE_EXPORT_NAME_SUBSTITUTION__":1};
+#elif ENVIRONMENT_MAY_BE_NODE || ENVIRONMENT_MAY_BE_SHELL
```

Comment:
Is this part of the change separate from only running `module_export_name_substitution` with closure?

Separately, perhaps add a comment here? I'm not sure why node/shell require this extra code.